vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -1088,7 +1087,7 @@ void NativeUIManager::findSubviewIn(
 
 void NativeUIManager::focus(int64_t reactTag) {
   if (auto shadowNode = static_cast<ShadowNodeBase *>(m_host->FindShadowNodeForTag(reactTag))) {
-    xaml::Input::FocusManager::TryFocusAsync(shadowNode->GetView(), winrt::FocusState::Programmatic);
+    xaml::Input::FocusManager::TryFocusAsync(shadowNode->GetView(), winrt::FocusState::Keyboard);
```

Comment:
Why would this be Keyboard instead of Programmatic? I assume this doesn't get called when the user hits the tab key.

Reply:
It's a workaround to render the border around a focused element - if Programmatic is specified, then there's no visual hint that the focus is on a given element.

Reply:
How do you know that's what they wanted though?  Programmatic seems more accurate.

Reply:
I assumed that we would like to indicate that focus has moved to an element, the same way it can be seen when tabbing through elements on the screen. I'm fine changing it, it just made sense to me to align those scenarios.

@asklar I think you've into the same issue in another project - what do you think?